pages/common/ned.md

```diff
@@ -0,0 +1,35 @@
+# ned
+
+> Is like grep, but with powerful replace capabilities, and unlike sed, as it isn't restricted to line oriented editing.
+
+- Recursively search starting in the current directory, ignoring case:
+
+`ned -iR '^[dl]og' .`
+
+- Search always showing colored output:
+
+`ned -c '^[dl]og' .`
```

Comment:
Same here.

Also, is there a long from of the `-c` option? We always prefer the longer options over shorter ones, as tey are more descriptive for new users.

Reply:
Done.